modules/plugin/coverage-multidim/netcdf/src/main/java/org/geotools/imageio/netcdf/utilities/NetCDFUtilities.java

```diff
@@ -1489,4 +1518,18 @@ public static ParameterBehaviour getParameterBehaviour(String parameter) {
             return ParameterBehaviour.DO_NOTHING;
         }
     }
+
+    private static void clearCache() {
+        URI_FORMAT_CACHE.clear();
+    }
+
+    /**
+     * Clear all the internal caches. Call this method if the datastore config has been updated or
+     * the NetCDF XML file has been modified.
+     */
+    public static void clearCaches() {
```

Comment:
Who's gonna use this? A listener for GeoServer "reset" button?

Reply:
Exactly